File: DAL/Entities/Alternative.cs
namespace DAL
{
    public class Alternative
    {
        public int AlternativeId { get; set; }

        public string AlternativeName { get; set; }


        public string DateCreated { get; set; }

        public string DateUpdated { get; set; }

        public Goal goal { get; set; }
    }
}

File: src/frontend/angular template/AHP-Angular/src/app/common/models/alternative.ts
type Priority={
    [key:number]:-9 | -7 | -5 | -3 | -1 | 1 | 3 | 5 | 7 | 9;
}
export class Alternative{
    AlternativeId:number;
    alternativeName:string;
    completed:boolean;
    GlobalPriority:Priority;
    order: number;

    constructor(name?: string) {
        this.alternativeName = name;
    }
}
